### DISCUSSION TITLE
Tumblr thumbnails

### DISCUSSION THREAD

**Discussion by T-prog3** [TumblThreeApp/TumblThree#302] [General]
I have the `Download video thumbnails` setting turned **_off_** and still have thumbnails downloaded on **_Tumblr_**. All of these are named with the ending `_frame1.jpg`, So they are easy to find and remove manually but should still be fixed.

**Comment by thomas694** [comment]
Thanks for reporting this. It seems a location was missed the last time.
I think it will have been fixed in the next version.

**Comment by thomas694** [comment]
The issue has been fixed and closed. You can still comment. Feel free to ask for reopening the issue if needed.
